docs/proxy/atlas/popular_tables.md

```diff
@@ -6,82 +6,44 @@ The required entity definitions for Atlas can be applied using [amundsenatlastyp
-        bulk_entities = list()
-        # Qualified Name (qn)
-        qn_entity = item.get('resource')
-        entity = client.entity_unique_attribute('Table', qualifiedName=qn_entity)
-        if entity.entity:
-            qn_metadata = f'{qn_entity}.metadata'
-
-            metadata = entity.entity.get("relationshipAttributes").get("metadata")
-
-            # Create Metadata if not exist already
-            if metadata:
-                metadata_guid = metadata["guid"]
-            else:
-                # Temp GUID
-                metadata_guid = -10000
-
-            # Add/Update Metadata and the popularityScore
-            bulk_entities.append(
-                {'typeName': 'table_metadata',
-                 'guid': metadata_guid,
-                 'attributes':
-                    {'qualifiedName': qn_metadata,
-                     'popularityScore': item.get("popularity_score"),
-                     'table': {'guid': entity.entity['guid'],
-                                      'typeName': entity.entity['typeName']}}
-                 }
-            )
-
-            for index, user in enumerate(item.get('users')):
-                qn_user, access_count = user
-                qn_reader = f'{qn_metadata}.{qn_user}.reader'
-                user_entity = client.entity_unique_attribute('User', qualifiedName=qn_user)
-                if user_entity.entity:
-                    user_guid = user_entity.entity["guid"]
-                else:
-                    user_guid = -20000 + index
-                    bulk_entities.append(
-                        {'typeName': 'User',
-                         'guid': user_guid,
-                         'attributes': {'qualifiedName': qn_user}
-                         }
-                    )
-                # Add/Update Reader object of user based on access counts
-                bulk_entities.append(
-                    {'typeName': 'Reader',
-                     'attributes': {'qualifiedName': qn_reader,
-                                    'count': access_count,
-                                    'entityMetadata': {'guid': metadata_guid},
-                                    'user': {'guid': user_guid}}
-                     }
-                )
-
-            if bulk_entities:
-                client.entity_bulk.create(data={"entities": bulk_entities})
-
+`Table` entity definition with `popularityScore` attribute [amundsenatlastypes==1.0.2](https://github.com/dwarszawski/amundsen-atlas-types/blob/master/amundsenatlastypes/schema/01_2_table_schema.json). 
+
+```json
+    {
+      "entityDefs": [
+        {
+          "name": "Table",
+          "superTypes": [
+            "DataSet"
+          ],
+          "attributeDefs": [
+            {
+              "name": "popularityScore",
+              "typeName": "float",
+              "isOptional": true,
+              "cardinality": "SINGLE",
+              "isUnique": false,
+              "isIndexable": false,
```

Comment:
why this value is `false` ? shouldn't this be indexable ?

Reply:
This is not used as a predicate but only for sorting. Introducing the index would probably improve the sort perfomance but may slow down write operations. Let's follow up on this discussion here https://github.com/dwarszawski/amundsen-atlas-types